interface/themes/oe-bootstrap.scss

```diff
@@ -1,82 +1,79 @@
-	margin-bottom: 10px!important;
-	padding: 0px 0px 15px 0px!important;
-}
-.btn-link.btn-cancel:focus, .btn-link.btn-cancel:hover {
-
-}
-.btn-link.btn-cancel {
-
-}
-.btn-link.btn-cancel.active, .btn-link.btn-cancel.focus,
-.btn-link.btn-cancel:active, .btn-link.btn-cancel:focus,
-.btn-link.btn-cancel:hover, .open>.dropdown-toggle.btn-link.btn-cancel {
-
-}
-.btn-separate-left,  .oe-opt-btn-separate-left{
-
-}
-.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle) {
-
-}
-.btn-group-pinch>.btn:nth-last-child(2):not(.dropdown-toggle), .oe-opt-btn-group-pinch>.btn:nth-last-child(2):not(.dropdown-toggle) {
-
-}
-.oe-custom-line{
-	padding-bottom:5px;
+    background-color: #F2F2F2!important;
+    margin-bottom: 10px!important;
+    padding: 0px 0px 15px 0px!important;
+}
+.btn-link.btn-cancel:focus, .btn-link.btn-cancel:hover {}
+.btn-link.btn-cancel {}
+.btn-link.btn-cancel.active, .btn-link.btn-cancel.focus, .btn-link.btn-cancel:active, .btn-link.btn-cancel:focus, .btn-link.btn-cancel:hover, .open>.dropdown-toggle.btn-link.btn-cancel {}
+.btn-separate-left, .oe-opt-btn-separate-left {}
+.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle) {}
+.btn-group-pinch>.btn:nth-last-child(2):not(.dropdown-toggle), .oe-opt-btn-group-pinch>.btn:nth-last-child(2):not(.dropdown-toggle) {}
+.oe-custom-line {
+    padding-bottom: 5px;
 }
 .oe-large {
-	display: inline-block;
+    display: inline-block;
 }
 .oe-small {
-	display: none;
+    display: none;
 }
 .oe-show-hide {
-	display: none;
+    display: none;
 }
-.oe-legend-radio{
-	font-size:12px;
-	padding:5px 5px 0px 0px;
+.oe-legend-radio {
+    font-size: 12px;
+    padding: 5px 5px 0px 0px;
 }
 .oe-bold-black {
-	font-weight: 700;
-	color: black;
+    font-weight: 700;
+    color: black;
 }
 .oe-payment-select {
-	width: 80px !important;
-	text-align: right;
-	font-size: 12px;
+    width: 80px !important;
+    text-align: right;
+    font-size: 12px;
 }
-.oe-ckbox-label{
-	padding-left: 30px;
-	font-weight: 500;
+.oe-ckbox-label {
+    padding-left: 30px;
+    font-weight: 500;
 }
-.oe-file-div{
-	padding:10px 15px;
+.oe-file-div {
+    padding: 10px 15px;
 }
-.oe-modal-dialog{
-	width:65% !important;
+.oe-modal-dialog {
+    width: 65% !important;
 }
 .oe-btn-circle {
-	width: 30px;
-	height: 30px;
-	text-align: center;
-	padding: 6px 0;
-	font-size: 12px;
-	line-height: 1.428571429;
-	border-radius: 15px;
-}
-/*above section is override bootstrap 3 for openemr and some new classes all prefixed with oe-*/
+    width: 30px;
+    height: 30px;
+    text-align: center;
+    padding: 6px 0;
+    font-size: 12px;
+    line-height: 1.428571429;
+    border-radius: 15px;
+}
+/* ---------------openemr 5 help file styling--------------- */
+.oe-help-heading {
+    color: #676666;
+    background-color: #E4E2E0;
+    border-color: #DADADA;
+}
+.oe-help-redirect {
+    color: #676666;
+}
+/* ---------------openemr 5 help file styling--------------- */
```

Comment:
Why is some help stuff here and some help stuff in the scss below?

Reply:
If these are overrides to the `oe-common` scss files they belong in the colors_base.scss. If they're truly independent, they can be left here. The goal is to eventually get to a consistent hierarchy of styles a la [SMACSS](https://smacss.com/) where the `core` files define the general rules that our layouts follow and the theme layers on top simply provide more specific selectors to augment the display of the components.

Reply:
hi @zbig01  ,
Just checking what your response and plan is on the above post?